forge-gui/res/cardsfolder/upcoming/navigation_orb.txt

```diff
@@ -2,7 +2,7 @@ Name:Navigation Orb
 ManaCost:3
 Types:Artifact
 A:AB$ ChangeZone | Cost$ 2 T Sac<1/CARDNAME> | Origin$ Library | Destination$ Library | ChangeType$ Land.Basic,Gate | ChangeNum$ 2 | RememberChanged$ True | Reveal$ True | Shuffle$ False | StackDescription$ SpellDescription | SubAbility$ DBChangeZone1 | SpellDescription$ Search your library for up to two basic land cards and/or Gate cards, reveal those cards, put one onto the battlefield tapped and the other into your hand, then shuffle.
-SVar:DBChangeZone1:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ ChangeType$ Land.IsRemembered,Gate.IsRemembered | ChangeNum$ 1 | Mandatory$ True | NoLooking$ True | SelectPrompt$ Select a card for the battlefield | Tapped$ True | Shuffle$ False | SubAbility$ DBChangeZone2 | StackDescription$ None
+SVar:DBChangeZone1:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.IsRemembered,Gate.IsRemembered | ChangeNum$ 1 | Mandatory$ True | NoLooking$ True | SelectPrompt$ Select a card for the battlefield | Tapped$ True | Shuffle$ False | SubAbility$ DBChangeZone2 | StackDescription$ None
 SVar:DBChangeZone2:DB$ ChangeZone | Origin$ Library | Destination$ Hand | ChangeType$ Land.IsRemembered,Gate.IsRemembered | Mandatory$ True | NoLooking$ True | SelectPrompt$ Select a card for your hand | StackDescription$ None | SubAbility$ DBCleanup
```

Comment:
so as I think about this, you must put one onto the battlefield

so if you pick only one, that goes to the battlefield

therefore, I like Mandatory in line 5 – I wonder if `ForgetChanged$ True` should also be added to that line

then line 6 could be `Defined$ Remembered` with no `ChangeType$` /`SelectPrompt$`/`Mandatory$`/`NoLooking$` but add `ConditionDefined$ Remembered | ConditionPresent$ Card | ForgetChanged$ True`

with the ForgetChanged you could also remove DBCleanup

Reply:
Made appropriate changes, functionality confirmed through testing. Thanks